src/router.tsx

```diff
@@ -5,6 +5,7 @@ import {
 } from 'react-router';
 import App from './App';
```

Comment:
Can we use Import alias 

```suggestion
import App from '@/App';
```

Reply:
Used import alias.
https://github.com/dhruvdhaduk-simform/ecommerce/pull/3/commits/2f6f67b99565de07a9e79ebd7f84df761c1a789c